--- ethminer-v0.18.0/libethash-cl/CMakeLists.txt.orig	2019-11-21 22:34:06.971449513 +0300
+++ ethminer-v0.18.0/libethash-cl/CMakeLists.txt	2019-11-21 22:34:26.511736637 +0300
@@ -24,8 +24,8 @@
 	# On macOS use system OpenCL library.
 	find_package(OpenCL REQUIRED)
 else()
-	hunter_add_package(OpenCL)
-	find_package(OpenCL CONFIG REQUIRED)
+	#hunter_add_package(OpenCL)
+	#find_package(OpenCL CONFIG REQUIRED)
 endif()
 
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
@@ -33,5 +33,5 @@
 
 add_library(ethash-cl ${SOURCES})
 target_link_libraries(ethash-cl PUBLIC ethcore ethash::ethash)
-target_link_libraries(ethash-cl PRIVATE OpenCL::OpenCL)
+target_link_libraries(ethash-cl PRIVATE OpenCL)
 target_link_libraries(ethash-cl PRIVATE Boost::filesystem Boost::thread)
